src/frontend/src/lib/components/receive/ReceiveAddress.svelte

```diff
@@ -44,9 +47,21 @@
 				/>
 			</div>
 
-			<output id="ic-wallet-address" class="break-all text-sm" data-tid={testId}>{address}</output>
+			{#if isNullish(address)}
```

Comment:
maybe invert the condition to be the same as the condition below?

Reply:
Good idea! Done. Thx